.eslintrc.js

```diff
@@ -128,6 +128,7 @@ module.exports = {
     ],
 
     // Unicorn rules
+    'unicorn/switch-case-braces': 'off',
```

Comment:
New rule. I don't like it. If anyone else does LMK.

Reply:
I wouldn't mind maybe using it with `['error', 'avoid']`:

> Only allow braces when there are variable declaration or function declaration which requires a scope.